### PR TITLE
[DOCS] Clarifies one of the installation steps for searchprovidercoveosearchservices

### PR DESCRIPTION
**Pull request by npushkarskii**
This PR clarifies one of the installation steps for the searchprovidercoveosearchservices extension.

### CODE REVIEW

File: coveo/searchprovidercoveosearchservices/README.md [hunk 1]
```diff
@@ -4,9 +4,8 @@
 
 1. Copy the `searchprovidercoveosearchservices` extension to the `hybris/bin/custom` directory of your project.
 
-1. Add `<extension name='searchprovidercoveosearchservices'/>` in your `localextensions.xml` file.
-
-1. Make sure to update your Maven settings to use the [Coveo Push API client library for Java](https://github.com/coveo/push-api-client.java).
+1. Add `<extension name='searchprovidercoveosearchservices'/>` in your `localextensions.xml` file. 
+The Coveo Push API client library for Java is now installed as part of the extension, see the `.jar` files in the `hybris/bin/custom/searchprovidercoveosearchservices/lib` directory.
```
Comment (npushkarskii) [inline coveo/searchprovidercoveosearchservices/README.md]:
```suggestion
The Coveo Push API client library for Java is now installed as part of the extension. 
See the `.jar` files in the `hybris/bin/custom/searchprovidercoveosearchservices/lib` directory.
```

File: coveo/searchprovidercoveosearchservices/README.md [hunk 2]
```diff
@@ -4,9 +4,9 @@
 
 1. Copy the `searchprovidercoveosearchservices` extension to the `hybris/bin/custom` directory of your project.
 
-1. Add `<extension name='searchprovidercoveosearchservices'/>` in your `localextensions.xml` file.
-
-1. Make sure to update your Maven settings to use the [Coveo Push API client library for Java](https://github.com/coveo/push-api-client.java).
+1. Add `<extension name='searchprovidercoveosearchservices'/>` in your `localextensions.xml` file. 
+The Coveo Push API client library for Java is now installed as part of the extension.
```
Comment (npushkarskii) [inline coveo/searchprovidercoveosearchservices/README.md]:
```suggestion
The Coveo Push API client library for Java is installed as part of the extension. 
```

**Review by jgwatkincoveo — DISMISSED** [review]

**Review by jgwatkincoveo — COMMENTED** [review]
this would automatically be updated once we update the readme in our development repo. The release process will automatically make the required changes.

**Comment by npushkarskii** [comment]
@jgwatkincoveo oh, I see... So I need to open a PR for https://github.com/coveo/cosap-commerce-connector ?

**Pull request closed without merging**